testplan/runnable/interactive/base.py

```diff
@@ -816,13 +751,13 @@ def reload_report(self):
                                     ].entries[param_index] = case[
                                         param_case.uid
                                     ]
-                                except KeyError:
+                                except (KeyError, IndexError):
                                     continue
                         else:
                             new_report[multitest.uid][suite.uid].entries[
                                 case_index
                             ] = suite[case.uid]
-                    except KeyError:
+                    except (KeyError, IndexError):
```

Comment:
We checked and this was breaking testcase removal, the reloader would not handle the IndexError that is thrown by lists, we only continued here originally upon KeyError.